<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="GLWidget" name="GLwidget" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>310</width>
        <height>310</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>220</width>
         <height>750</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QGroupBox" name="renderingBox">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>130</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Rendering</string>
          </property>
          <widget class="QCheckBox" name="smoothShadeCheckBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>131</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Smooth Shade</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="wireframeCheckBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>151</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Show Wireframe</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="algorithmsBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>Algorithms</string>
          </property>
          <widget class="QPushButton" name="findModeButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>99</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>Find Modes</string>
           </property>
          </widget>
          <zorder>findModeButton</zorder>
          <zorder>groupBox</zorder>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>Internal Modes</string>
          </property>
          <widget class="QLabel" name="showModeLabel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>121</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Show mode:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="showModelEdit">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>30</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="parameterBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>400</height>
           </size>
          </property>
          <property name="title">
           <string>Parameters</string>
          </property>
          <widget class="QGroupBox" name="physicalBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>191</width>
             <height>151</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Physical</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <widget class="QLabel" name="youngsModulusLabel">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>30</y>
              <width>121</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Young's Modulus:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="youngsModulusEdit">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>30</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="poissonRatioLabel">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>60</y>
              <width>121</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Poisson Ratio:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="poissonRatioEdit">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>60</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="thicknessLabel">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>90</y>
              <width>121</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Thickness:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="thicknessEdit">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>90</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="densityLabel">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>120</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Density:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="densityEdit">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>120</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVViewer">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_OBJ"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionReset_Camera"/>
    <addaction name="actionTake_Screenshot"/>
    <addaction name="actionExport_OBJ"/>
   </widget>
   <addaction name="menuVViewer"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReset_Camera">
   <property name="text">
    <string>Reset Camera</string>
   </property>
  </action>
  <action name="actionTake_Screenshot">
   <property name="text">
    <string>Take Screenshot</string>
   </property>
  </action>
  <action name="actionExport_OBJ">
   <property name="text">
    <string>Export 3D Mesh</string>
   </property>
  </action>
  <action name="actionImport_OBJ">
   <property name="text">
    <string>Import OBJ</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
